doc/chomp_interface/scripts/collision_scene_test_1.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+import rospy
+from moveit_commander import RobotCommander, PlanningSceneInterface
+import geometry_msgs.msg
+import time
+
+
+class CreateScene(object):
+
+    def __init__(self):
+        self._scene = PlanningSceneInterface()
+
+        # clear the scene
+        self._scene.remove_world_object()
+
+        self.robot = RobotCommander()
+
+        # pause to wait for rviz to load
+        rospy.sleep(4)
+
+        floor_pose = [0, 0, -1.12, 0, 0, 0, 1]
+        floor_dimensions = [0, 0, 0.0]
+
+        box1_pose = [0.20, 0.50, 0.25, 0, 0, 0, 1]
+        box1_dimensions = [0.2, 0.2, 0.5]
+
+        box2_pose = [-0.55, -0.55, 0, 0, 0, 0, 1]
+        box2_dimensions = [0.25, 0.25, 1.75]
+
+        box3_pose = [0.5, -0.55, 0.14, 0, 0, 0, 1]
+        box3_dimensions = [0.28, 0.28, 0.22]
+
+        box4_pose = [-0.4, 0.4, 0.5, 0, 0, 0, 1]
+        box4_dimensions = [0.25, 0.25, 1.1]
+
+        self.add_box_object("floor", floor_dimensions, floor_pose)
+        self.add_box_object("box1", box1_dimensions, box1_pose)
+        self.add_box_object("box2", box2_dimensions, box2_pose)
+        self.add_box_object("box3", box3_dimensions, box3_pose)
+        self.add_box_object("box4", box4_dimensions, box4_pose)
+
+    def add_box_object(self, name, dimensions, pose):
+        p = geometry_msgs.msg.PoseStamped()
+        p.header.frame_id = self.robot.get_planning_frame()
+        p.header.stamp = rospy.Time.now()
+        p.pose.position.x = pose[0]
+        p.pose.position.y = pose[1]
+        p.pose.position.z = pose[2]
+        p.pose.orientation.x = pose[3]
+        p.pose.orientation.y = pose[4]
+        p.pose.orientation.z = pose[5]
+        p.pose.orientation.w = pose[6]
+
+        self._scene.add_box(name, p, (dimensions[0], dimensions[1], dimensions[2]))
+	print "============ Waiting while RVIZ displays the scene with four obstacles..."
+        rospy.sleep(1)
```

Comment:
Please don't use sleep. See the move_group_python_tutorial for explicitly waiting for the ps to update

Reply:
I removed this particular sleep usage, but I still got the sleep usage over [here](https://github.com/raghavendersahdev/moveit_tutorials/blob/kinetic-devel/doc/chomp_interface/scripts/collision_scene_example_sparse.py#L17). to show the user some sort of terminal output while loading the scene.The [move_group_python_tutorial](http://docs.ros.org/indigo/api/moveit_tutorials/html/doc/pr2_tutorials/planning/scripts/doc/move_group_python_interface_tutorial.html) for pr2 also used sleep like : 
![selection_025](https://user-images.githubusercontent.com/5405862/41176531-ca9a9e50-6b2e-11e8-96ea-eebddf0aca50.png)
![selection_026](https://user-images.githubusercontent.com/5405862/41176536-cce0b5b4-6b2e-11e8-9b9c-8a144d41c78b.png)